sanity-io/sanity
# chore: use pnpm as package manager

### Description
This migrates this repo to use pnpm instead of (legacy) yarn 1.x


### Testing
- remove all node_modules etc.
- run pnpm install
- verify that things still works (test studios, scripts, etc)

### Notes for release

n/a - not user facing
